### PR TITLE
Convert toggle functions to TypeScript

### PR DESCRIPTION
**Pull request by iantrich**
Note: These changes are not really "testable" until we resolve the longPress directive issue that was introduced with the upgrading of lit-html

### CODE REVIEW

File: src/panels/lovelace/common/entity/turn-on-off-entity.ts [hunk 1]
```diff
@@ -17,4 +22,4 @@ export default function turnOnOffEntity(hass, entityId, turnOn = true) {
   }
 
   hass.callService(serviceDomain, service, { entity_id: entityId });
```
Comment (balloob) [inline src/panels/lovelace/common/entity/turn-on-off-entity.ts]: we should return this promise so we can await until command is completed.

File: src/panels/lovelace/common/entity/toggle-entity.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { STATES_OFF } from "../../../../common/const";
+import { turnOnOffEntity } from "./turn-on-off-entity";
+import { HomeAssistant } from "../../../../types";
+
+export const toggleEntity = (hass: HomeAssistant, entityId: string): void => {
+  const turnOn = STATES_OFF.includes(hass.states[entityId].state);
+  turnOnOffEntity(hass, entityId, turnOn);
```
Comment (balloob) [inline src/panels/lovelace/common/entity/toggle-entity.ts]: let's return this.

File: src/panels/lovelace/cards/hui-picture-glance-card.js [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+import computeDomain from "../../../common/entity/compute_domain";
+
+const STATES_OFF = new Set(["closed", "locked", "not_home", "off"]);
+
+/*
+ * @appliesMixin EventsMixin
+ * @appliesMixin LocalizeMixin
+ * @appliesMixin NavigateMixin
+ */
+class HuiPictureGlanceCard extends NavigateMixin(
```
[the remark this answers is not in the document]
Comment (iantrich) [reply src/panels/lovelace/cards/hui-picture-glance-card.js]: shoot!

**Comment by iantrich** [comment]
I don't have time right now to resolve this merge conflict. Will take a look tonight, hopefully.

**Pull request closed without merging**